lib/produce/itunes_connect.rb

```diff
@@ -10,12 +10,19 @@ class ItunesConnect < FastlaneCore::ItunesConnect
 
     def run(config)
       @config = config
+      @wildcard_bundle = config[:bundle_identifier].end_with?("*")
+
+      if @wildcard_bundle && @config[:bundle_identifier_suffix].nil? 
+        raise "Bundle id has wildcard in it. You must specify a valid bundle_identifier_suffix".red
+      end
```

Comment:
Please remove this and add a question [there](https://github.com/dral3x/produce/blob/wildcard-app-id/lib/produce/config.rb#L8), so so `produce` will ask user for bundle suffix if not given.